.github/workflows/build.yaml

```diff
@@ -38,12 +38,20 @@ jobs:
           distribution: 'zulu'
           java-version: 11
 
-      # Clang is seemingly already installed on Ubuntu but fails to build the project with errors
-      # like "file not found" for `#include <string>`. Explicitly (re?)installing fixes things.
-      - run: sudo apt-get install clang
+      - name: Build native library (Linux)
         if: matrix.os == 'ubuntu-latest'
-
-      - name: Build native library
+        run: |
+          cp -a . ../zipline-dockerbuild
+          docker run \
+            --mount src=`pwd`/../zipline-dockerbuild,target=/zipline,type=bind \
+            amd64/centos:7 \
+            /zipline/.github/workflows/build-linux-amd64.sh
+          mkdir -p zipline/src/jvmMain/resources/jni/amd64
+          cp -v ../zipline-dockerbuild/build/jni/amd64/libquickjs.* zipline/src/jvmMain/resources/jni/amd64/
+          docker system prune -a -f
```

Comment:
Is this needed on CI? I would think a `--rm` on the `run` command (or really, doing nothing) would be fine.

Reply:
I was thrashing trying to figure out what was causing the Kotlin daemon to get stuck. I think it’s cause I’d had the project directory in a Docker mount when Kotlin was running, and that might have broken special files?